presto-raptor/src/main/java/com/facebook/presto/raptor/storage/ShardCompactionManager.java

```diff
@@ -23,6 +23,7 @@
 import com.facebook.presto.raptor.metadata.TableMetadata;
 import com.facebook.presto.spi.NodeManager;
 import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.util.CpuTimer;
```

Comment:
In my opinion it's not worth making Raptor an internal plugin just to reuse this. I would just copy it out of presto-main